app/diagnostics.go

```diff
@@ -298,6 +298,7 @@ func (a *App) trackConfig() {
 		"driver_name":                 *cfg.SqlSettings.DriverName,
 		"trace":                       cfg.SqlSettings.Trace,
 		"max_idle_conns":              *cfg.SqlSettings.MaxIdleConns,
+		"max_lifetime_conns":          *cfg.SqlSettings.MaxLifetimeConns,
```

Comment:
Can we call this `conn_max_lifetime_milliseconds` to match the `ConnMaxLifetime` used in [database/sql](https://golang.org/src/database/sql/sql.go#L817) and clearly identify the units in question? (Converging on milliseconds to match the granularity used similar settings, like `ClusterSettings.IdleConnTimeoutMilliseconds`).